spec/features/admin/admin_login_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+feature "an admin logs in" do 
+  let!(:admin) {User.create({fullname: "Fred Durst", display_name: "Dbag55", email: "Buckethat@gmail.com", password: "password", password_confirmation: "password", phone: "303-345-1234", role: 1})}
```

Comment:
Line is too long. [197/80]<br>Redundant curly braces around a hash parameter.<br>Space missing inside {.<br>Space missing inside }.<br>Space inside { missing.<br>Space inside } missing.